src/components/ItemTile/ItemTile.module.scss

```diff
@@ -72,6 +72,9 @@
     align-items: stretch;
     height: 50vh;
   }
+  .container:last-child {
+    margin-bottom: 222px;
```

Comment:
Nie powinniśmy w niezależnych komponentach ustawiać takich wartości, komponenty powinny być jak najbardziej niezależne. Można to ustawić bezpośrednio w widoku, który kodujemy

Reply:
już usunięte